pkg/provider/azure_loadbalancer.go

```diff
@@ -585,6 +585,7 @@ func (az *Cloud) getServiceLoadBalancer(service *v1.Service, clusterName string,
 	}
 
 	// check if the service already has a load balancer
+	var shouldChange bool
```

Comment:
nit: rename to 'shouldChangeLB' for clarity

Reply:
done